cypress/e2e/Navigation.cy.js

```diff
@@ -0,0 +1,157 @@
+import MainMenu from "../../cypress/PageObject/MainMenu";
+
+const menu= new MainMenu()
+
+describe('Navigation', () => {
+    
+
+    it ('Back_to_products_by_name',()=> {
+
+        cy.login_standard()
+
+        menu.get_item_title_link('0').click()
+        cy.url().should('eq','https://www.saucedemo.com/inventory-item.html?id=0')
+        menu.back_to_products()
+        menu.url_inventory_is_good()
+
+    })
+
+    it ('Back_to_products_by_image',()=> {
+
+        cy.login_standard()
+        
+        cy.get('[id="item_1_img_link"]').click()
+        cy.url().should('eq','https://www.saucedemo.com/inventory-item.html?id=1')
```

Comment:
I suggest to move this to some Page Object

Reply:
done